notebooks/Example_Fitting_a_Linear_Model.ipynb

```diff
@@ -39,14 +39,12 @@
    "outputs": [
```

Comment:
<p>The arguments should be <code>hdi_lower </code>and <code>hdi_upper </code>here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/JuBiotech/calibr8/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>